csrankings-t.csv

```diff
@@ -939,6 +939,7 @@ Travis J. Desell,Rochester Institute of Technology,http://http://tdesell.cs.und.
 Trent Jaeger,Univ. of California - Riverside,http://www.trentjaeger.com,JS8aznIAAAAJ
 Trevor Alexander Brown,University of Waterloo,http://www.cs.toronto.edu/~tabrown,MaSuaDkAAAAJ
 Trevor Anthony Cohn,University of Melbourne,http://people.eng.unimelb.edu.au/tcohn,FCom398AAAAJ
+Trevor Bonjour,Univ. of California - San Diego,https://profiles.ucsd.edu/trevor.bonjour,FeINO6AAAAAJ

```

Comment:
Can you clarify? Listed as LECT (https://profiles.ucsd.edu/trevor.bonjour).

Reply:
They are tenure-track teaching professor. UCSD has a teaching line equivalent to faculty line, just different promotion metrics.